index.html

```diff
@@ -3,11 +3,62 @@
 <head>
     <title>Academics | NUS Computing</title>
```

Comment:
Add the following lines above the `<title>` tag
```html
	<meta charset="utf-8" />
	<meta name="viewport" content="width=device-width, initial-scale=1.0" />
```
This will show the mobile view on mobile sized screens (right now it doesn't work that well)